types/src/das.rs

```diff
@@ -0,0 +1,33 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+/// Represents a worker responsible for DAS (Data Availability Sampling) jobs.
+pub struct DasWorker {
+    /// Type of the job the worker is performing.
+    pub job_type: String,
+    /// Current progress of the job.
+    pub current: u64,
+    /// The starting height of the job.
+    pub from: u64,
+    /// The ending height of the job.
+    pub to: u64,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+/// Contains statistics related to the DAS process.
+pub struct SamplingStats {
```

Comment:
can you reorder doc-comment and derive directive? I think everywhere else we keep doc comments first, and then derive